lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,6 +825,14 @@ def _download_direct_download_link(self, file_url, copy_file_path):
             with closing(online_file):
                 with open(copy_file_path, 'wb') as output:
                     shutil.copyfileobj(online_file, output)
+            # check first 5 lines of file content
+            with open(copy_file_path) as copied_file:
+                for line in islice(copied_file, 5):
+                    if line.lower().find('html') != -1:
+                        raise ValueError("Undownable File.\n" + 
+                            "Please make sure file is publicly accessible\n" +
+                            "File URL: %s" % file_url)
```

Comment:
@MrCreosote I am trying to prevent user to input something like "www.google.com".Or in the case that the link is not publicly accessible, cloud API will return a webpage instead of the actual file. The code will continue executing till ValidateFASTQ find out it's not a fastq file format. 

In this downloader, downloader link should just be a fastq file. It shouldn't be a html file I think.